src/main/scala/com/softwaremill/realworld/Main.scala

```diff
@@ -22,7 +23,7 @@ object Main extends ZIOAppDefault:
           )
         )
         .options
-    val app: HttpApp[Any, Throwable] = ZioHttpInterpreter(serverOptions).toHttp(Endpoints.all)
+    val app: HttpApp[Any, Throwable] = ZioHttpInterpreter(serverOptions).toHttp(endpoints)
```

Comment:
nit: it is actually about the line #33 but I wasn't able to add comment there 😄 so here we go: what I like about stubs generated by `adopt-tapir` is that they inform in the console where if and where the `docs` are available IOW one doesn't have to guess it ;)

Reply:
I've updated the message.

Reply:
Where does the `endpoints` value come from?